Source/Controls/CommandBar/CommandBarButton.ts

```diff
@@ -1,14 +1,11 @@
 // Copyright (c) Dolittle. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-import { customElement, containerless, bindable } from 'aurelia-framework';
+import { containerless, bindable } from 'aurelia-framework';
 
-@customElement('command-bar-button')
 @containerless()
 export class CommandBarButton {
-    @bindable icon: any;
-    @bindable label: any;
-    @bindable linkUrl: any;
-    constructor(){
-
-    }
+  @bindable label: String = '';
+  @bindable linkUrl: String = '';
```

Comment:
This is very limiting to specific behavior of navigating somewhere and something the *CommandBarButton* shouldn’t really have the responsibility for. This should be decided on by the consuming view+ViewModel. So typically a click event would be what we’d expect and then let whoever is consuming and using this decide wether or not its a navigational action or something else.

Reply:
So. this component has been hanging around for a while, and was made for a different purpose at the time that it actually should be aimed for. Perhaps we should remove linkUrl as you say and add the callback function right away. Why I created a task for it was mostly that this branch has been running for a couple of weeks now, and wanted to get this done and then add the correct behavior afterwards. But I can do it now.

How about a different button; CommandBarLinkButton ?